src/patternfly/components/Select/select.scss

```
@@ -111,6 +111,14 @@
   // Match within a menu item when using typeahead
   --pf-c-select__menu-item--match--FontWeight: var(--pf-global--FontWeight--bold);
 
+  // Menu group padding
+  --pf-c-select__menu-group--m-padding--PaddingTop: var(--pf-c-select__menu--PaddingBottom);
+  --pf-c-select__menu-group--m-padding--PaddingRight: var(--pf-c-select__menu-item--PaddingRight);
+  --pf-c-select__menu-group--m-padding--PaddingBottom: var(--pf-c-select__menu--PaddingBottom);
+  --pf-c-select__menu-group--m-padding--PaddingLeft: var(--pf-c-select__menu-item--PaddingLeft);
+  --pf-c-select__menu-group--m-padding--not--first-of-type--PaddingTop: var(--pf-global--spacer--md);
```

```suggestion
  --pf-c-select__menu-group--m-padding--not-first-of-type--PaddingTop: var(--pf-global--spacer--md);
```
do you need the `--` here?

---

Yeah, because the selector is `:not(:last-of-type)`, the second `-` denotes a pseudo selector.

@mcoker @christiemolloy wdyt?

---

Agree, it's 2 pseudo selectors.

---

Yep, agreed